docs/src/pages/[platform]/connected-components/authenticator/customization/customization.override-function-calls.react-native.mdx

```diff
@@ -31,32 +31,32 @@ Each `handle*` function will return the neccessary values you'll need to make th
   </TableHead>
   <TableBody>
     <TableRow>
-      <ResponsiveTableCell label="Function Call">`Auth.signUp`</ResponsiveTableCell>
+      <ResponsiveTableCell label="Function Call">`signUp()`</ResponsiveTableCell>
```

Comment:
Let's remove the parans on the function names, that is not a pattern that has been used in the UI docs previously, nor does it provide the correct input for the functions